doc/source/structures/celestial_bodies/atmosphere.rst

```diff
@@ -104,6 +97,23 @@ A Structure closely tied to :struct:`Body` A variable of type :struct:`Atmospher
 
     The altitude at which the atmosphere is "officially" advertised as ending. (actual ending value differs, see below).
 
+Deprecated Suffix
+-----------------
+
+.. attribute:: Atmosphere:SCALE
+
+    :type: :ref:`scalar <scalar>`
+    :access: Get only
+
+    A math constant plugged into a formula to find atmosphere density.
+
+    .. note::
+
+        .. deprecated:: 0.17.2
+
+            To get the same functionality, you must use :attr:`Vessel:PATCHES`  which is a suffix of the :struct:`Vessel` itself.
```

Comment:
That's exactly what happened.  Both `TERMVELOCITY` and `SCALE`  were meant to have the same note because they were both removed for the same reason, but I never updated `SCALE` after I copy and pasted `Vessel:PATCHES`, which I used as a framework for the formatting.   I pushed a new commit that fixes this.